schema.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE animals (
+    ID INT GENERATED ALWAYS AS IDENTITY,
+    NAME VARCHAR NOT NULL,
+    DATE_OF_BIRTH DATE NOT NULL,
+    ESCAPE_ATTEMPTS INT NOT NULL,
+    NEUTERED BOOLEAN NOT NULL,
+    WEIGHT_KG DECIMAL NOT NULL
+);
+ALTER TABLE animals 
+ADD COLUMN species VARCHAR;
+
+CREATE TABLE owners (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    full_name VARCHAR NOT NULL,
+    age INT,
+    PRIMARY KEY (id)
+);
+CREATE TABLE species (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR NOT NULL,
+    PRIMARY KEY (id)
+);
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals 
+ADD COLUMN species_id INT;
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id);
+
+ALTER TABLE animals 
+ADD COLUMN owner_id INT;
```

Comment:
- [ ] Good work adding a column to the table animals, as it's requested in the requirements. Kindly add the references from this new column to `owners(id)`. It's follow the same patter as you did with the `species(id)` **REFERENCES**